docs/quickstart.md

```diff
@@ -164,40 +164,38 @@ export KOLENA_TOKEN="********"
 We can now create test suites using the provided seeding script:
 
 ```shell
-poetry run python3 age_estimation/seed_test_suite.py
+poetry run python3 object_detection_2d/seed_test_suite.py
 ```
 
 After this script has completed, we can visit the [:kolena-test-suite-16: Test Suites](https://app.kolena.io/redirect/testing)
 page to view our newly created test suites.
 
-In this `age_estimation` example, we've created test suites stratifying the LFW dataset (which is stored as a CSV in
-S3) into test cases by age, estimated race, and estimated gender.
+In this `object_detection_2d` example, we've created test suites stratifying the [COCO](https://cocodataset.org/#overview) 2014 validation set (which is stored as a CSV in
+S3) into test cases by brightness and bounding box size. In this example will be looking at the following labels:
+
+`["bicycle", "car","motorcycle", "bus", "train", "truck", "traffic light", "fire hydrant", "stop sign" ]`
 
 ## Test a Model
 
-After we've created test suites, the final step is to test models on these test suites. The `age_estimation` example
-provides the `ssrnet` model for this step:
+After we've created test suites, the final step is to test models on these test suites. The `object_detection_2d` example
+provides the following models to choose from `{yolo_r,yolo_x,mask_rcnn,faster_rcnn,yolo_v4s,yolo_v3}` for this step:
```

Comment:
Should we add space between each model name?

Reply:
done